server/routes/v1/tests/user.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Request, Response } from "express";
+
+import db from "../../../models/index";
+import { signUpHandler, signInHandler } from "../user";
+
+function mockRequest(data: any) {
+  return { body: data };
+}
+
+function mockResponse() {
+  const res: any = {};
+  res.status = jest.fn().mockReturnValue(res);
+  res.json = jest.fn().mockReturnValue(res);
+  res.cookie = jest.fn().mockReturnValue(res);
+  return res;
+}
+
+beforeAll(async () => {
+  await db.sequelize.sync({ force: true });
```

Comment:
There is a helper function that does this, check the testHelpers file in user models test